ztp/source-crs/AmqSubscriptionNS.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: amq-router
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
Note that the amq pod (update needed in its codebase) will need the corresponding annotation in order to be allocated to the management cores:
` target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'`

Reply:
Create a Jira for AMQ team to track this requirement 
https://issues.redhat.com/browse/ENTMQIC-3265